apps/meteor/app/livechat/server/methods/getAnalyticsOverviewData.ts

```diff
@@ -18,6 +19,7 @@ declare module '@rocket.chat/ui-contexts' {
 
 Meteor.methods<ServerMethods>({
 	async 'livechat:getAnalyticsOverviewData'(options) {
+		methodDeprecationLogger.method('livechat:getAnalyticsOverviewData', '7.0.0', ' Use "livechat/analytics/overview" instead.');
```

Comment:
Move this to another PR :)

(Both these should be on a changeset)

Reply:
Moved it here -> https://github.com/RocketChat/Rocket.Chat/pull/30421